recipes/peddy/meta.yaml

```diff
@@ -1,17 +1,16 @@
 {% set name = "peddy" %}
-{% set version = "0.3.6a" %}
-{% set revision = "62bed8f" %}
-{% set sha256hash = "f50dfec6b6f12497fbbb0a92d288a6d07cba3f6548c14fd72b93a0b055d347d1" %}
+{% set version = "0.4.1" %}
+{% set sha256hash = "f4b384ebd866756015ef3c2ba6f206e158df6a26cc6d4c8d2afa474325c7feda" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: peddy-{{ revision }}.tar.gz
-  url: https://github.com/brentp/peddy/archive/{{ revision }}.tar.gz
-  #fn: peddy-{{ version }}.tar.gz
-  #url: https://github.com/brentp/peddy/archive/v{{ version }}.tar.gz
+  #fn: peddy-{{ revision }}.tar.gz
+  #url: https://github.com/brentp/peddy/archive/{{ revision }}.tar.gz
```

Comment:
I think would be great if we just delete this commented code :)

Another thing, this package is a pure python package, right?